app/src/main/java/org/oppia/android/app/topic/revisioncard/PracticeTabModule.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.android.app.topic
+
+import dagger.Module
+import dagger.Provides
+
+/** Provides dependencies corresponding to the practice tab. */
+@Module
+class PracticeTabModule {
```

Comment:
I think by mistake this file shifted to `revisioncard` package.

Reply:
Nice catch. Fixed (will be visible in the next push to this branch).